ImperatorToCK3/Source/Imperator/Countries/Country.cpp

```diff
@@ -25,13 +25,13 @@ void ImperatorWorld::Country::registerKeys()
 		flag = flagStr.getString();
 	});
 	registerKeyword("color", [this](const std::string& unused, std::istream& theStream) {
-		color1 = commonItems::Color::Factory::getColor(theStream);
+		color1 = commonItems::Color::Factory{}.getColor(theStream);
```

Comment:
mmyeah. If you get "white" somewhere you will have to use our brand new smart color factory for this.

Also be sure *all* colors you read like this go into std::optional<Color>. Otherwise you can't distinguish an unset color from a black one.

Reply:
Fortunately they don't seem to use named colors here, just RGB.